src/less-style/question-props.less

```diff
@@ -157,3 +157,52 @@
   padding: 0 2em;
   margin-bottom: 2em;
 }
+
+.fake-input {
```

Comment:
What do you think of not using the word "fake" in the name since it's not fake? Maybe `input` or `fd-input`? Same with "fake-textarea" below.